abel/transform.py

```diff
@@ -19,7 +19,7 @@
 def transform(IM,
               direction='inverse', method='three_point', center='none',
               symmetry_axis=None, use_quadrants=(True, True, True, True),
-              angular_integration=False,
+              symmetrize='average', angular_integration=False,
```

Comment:
Do we want to call this `symmetrize_method`? Calling it just `symmetrize` seems to imply that it's a boolean.